bundles/org.openhab.binding.modbus.lambda/src/main/resources/OH-INF/thing/boiler-channel-types.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="boiler-error-number-type">
```

Comment:
both prefix `boiler-` and suffix `-type` can be removed, applies to all channel types.